service/src/main/java/bio/terra/workspace/service/resource/referenced/cloud/gcp/gcsobject/ReferencedGcsObjectResource.java

```diff
@@ -161,51 +178,21 @@ public boolean checkAccess(FlightBeanBag context, AuthenticatedUserRequest userR
   public Builder toBuilder() {
     return builder()
         .bucketName(getBucketName())
-        .fileName(getObjectName())
-        .cloningInstructions(getCloningInstructions())
-        .description(getDescription())
-        .name(getName())
-        .resourceId(getResourceId())
-        .workspaceId(getWorkspaceId())
-        .resourceLineage(getResourceLineage());
+        .objectName(getObjectName())
+        .resourceCommonFields(getWsmResourceFields());
   }
 
   public static Builder builder() {
     return new Builder();
   }
 
   public static class Builder {
-    private CloningInstructions cloningInstructions;
+    private WsmResourceFields resourceCommonFields;
```

Comment:
I was a bit confused that some place have `resourceCommonFields` and some have `wsmResourceFields`. Can they be the same (I don't care which one). Same for rest of PR.